_inc/lib/core-api/class.jetpack-core-api-module-endpoints.php

```diff
@@ -1503,7 +1503,7 @@ private function akismet_is_active_and_registered() {
 	 *     @type string $date Date range to restrict results to.
 	 * }
 	 *
-	 * @return int|string Number of spam blocked by Akismet. Otherwise, an error message.
+	 * @return WP_Error|WP_HTTP_Response|WP_REST_Response Stats information relayed from WordPress.com
```

Comment:
I think you may need a period at the end of the line to make the linter happy?

```suggestion
	 * @return WP_Error|WP_HTTP_Response|WP_REST_Response Stats information relayed from WordPress.com.
```